isis/src/hayabusa2/apps/hyb2pds4gen/main.cpp

```diff
@@ -18,9 +20,32 @@
 using namespace std;
 using namespace Isis;
 
+Pvl toPvl(PvlGroup &container) { 
+  Pvl newPvl;
+  std::stringstream buffer;
+  buffer << container << std::endl; 
+  buffer >> newPvl;
+  return newPvl; 
+}
+
+Pvl toPvl(PvlObject &container) { 
+  Pvl newPvl;
+  std::stringstream buffer;
+  buffer << container << std::endl; 
+  buffer >> newPvl;
+  return newPvl; 
+}
```

Comment:
Duplicate method

Reply:
input Type is different which actually matters, that being said, I'm no longer using one of them so I should get rid of that one.

Reply:
Oops my bad. I read the input as the same.